src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public static Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setColor(ColorSupplier.getRandomColor());
```

Comment:
Create an instance of ColorSupplier and use its method to get a random color, instead of using a static method.